crates/collab_ui/src/collab_ui.rs

```diff
@@ -122,5 +122,6 @@ fn notification_window_options(
         display_id: Some(screen.id()),
         fullscreen: false,
         window_background: WindowBackgroundAppearance::default(),
+        app_id: Some("Zed".to_owned()),
```

Comment:
Looking at the ids for Firefox and Konsole, the correct one would be `dev.zed.zed`.

Reply:
@jakobhellermann Looking at Flathub, it seems that most programs actually follow the reverse DNS convention, with the program name starting with an upper-case letter (like in the documentation you linked). Additionally, someone is working on a Flatpak package (https://github.com/someone13574/zed/tree/flatpak) and they're also using `dev.zed.Zed`, so you should probably change the `app_id` field to match that.